.github/workflows/linters.yml

```diff
@@ -0,0 +1,49 @@
+name: Linters
+
+on:
+  pull_request:
+    branches: 
+     - "**"
+     
+env:
+  TARGET_BRANCH: ${{ github.base_ref }}
+
+jobs:
+  run-linters:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Run npm install
+      run: npm install --save-dev
+
+    - name: Fetch target branch
+      run: git fetch -n origin ${TARGET_BRANCH}
+            
+    - name: Run phpcs linter 
+      run: |
+        FILES_DIFF=$(git diff origin/${TARGET_BRANCH} --no-commit-id --name-only -r -- '*.php')
```

Comment:
@erawat `--no-commit-id` and `-r` are not valid params of the `git diff` command. Please remove it.